packages/vaadin-overlay/src/vaadin-overlay-position-mixin.js

```diff
@@ -80,25 +81,57 @@ export const PositionMixin = (superClass) =>
     static get observers() {
       return [
         '__positionSettingsChanged(positionTarget, horizontalAlign, verticalAlign, noHorizontalOverlap, noVerticalOverlap)',
-        '__overlayOpenedChanged(opened)',
+        '__overlayOpenedChanged(opened, positionTarget)',
```

Comment:
Now when we added `positionTarget` here, we can remove it from the above observer which also calls `_updatePosition()`. Not a blocker.

Reply:
Done.